app/javascript/mastodon/components/video_player.js

```diff
@@ -119,11 +123,15 @@ export default class VideoPlayer extends React.PureComponent {
       </div>
     );
 
-    let expandButton = (
-      <div className='status__video-player-expand'>
-        <IconButton overlay title={intl.formatMessage(messages.expand_video)} icon='expand' onClick={this.handleExpand} />
-      </div>
-    );
+    let expandButton = '';
+
+    if (this.context.router) {
```

Comment:
Afaik there is no good way to simulate an anchor link click without an anchor link. So I can't just make clicking the video open it in a new tab/window/whatever.

Also, presence of router context is currently a good check for whether we're inside `<UI />` or not.